2018-07-08 11:05:04,207 ERROR: Exception on /view_stories [GET] [in /home/mark/anaconda3/envs/flask/lib/python3.6/site-packages/flask/app.py:1560]
Traceback (most recent call last):
  File "/home/mark/anaconda3/envs/flask/lib/python3.6/site-packages/flask/app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/mark/anaconda3/envs/flask/lib/python3.6/site-packages/flask/app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/mark/anaconda3/envs/flask/lib/python3.6/site-packages/flask/app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/mark/anaconda3/envs/flask/lib/python3.6/site-packages/flask/_compat.py", line 33, in reraise
    raise value
  File "/home/mark/anaconda3/envs/flask/lib/python3.6/site-packages/flask/app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/mark/anaconda3/envs/flask/lib/python3.6/site-packages/flask/app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/mark/PythonFlaskVertica/app/routes.py", line 328, in view_stories
    return render_template('view_stories.html', title='View Stories', all_stories = all_stories)
  File "/home/mark/anaconda3/envs/flask/lib/python3.6/site-packages/flask/templating.py", line 134, in render_template
    context, ctx.app)
  File "/home/mark/anaconda3/envs/flask/lib/python3.6/site-packages/flask/templating.py", line 116, in _render
    rv = template.render(context)
  File "/home/mark/anaconda3/envs/flask/lib/python3.6/site-packages/jinja2/asyncsupport.py", line 76, in render
    return original_render(self, *args, **kwargs)
  File "/home/mark/anaconda3/envs/flask/lib/python3.6/site-packages/jinja2/environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "/home/mark/anaconda3/envs/flask/lib/python3.6/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/mark/anaconda3/envs/flask/lib/python3.6/site-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "/home/mark/PythonFlaskVertica/app/templates/_stories.html", line 2, in template
    <div class="col-xs-2"><a href="{{ url_for('view_story_detail', id=story_detail.id)) }}">
  File "/home/mark/anaconda3/envs/flask/lib/python3.6/site-packages/jinja2/environment.py", line 1005, in render
    return concat(self.root_render_func(self.new_context(vars)))
  File "/home/mark/PythonFlaskVertica/app/templates/view_stories.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/home/mark/PythonFlaskVertica/app/templates/base.html", line 1, in top-level template code
    {% extends 'bootstrap/base.html' %}
  File "/home/mark/anaconda3/envs/flask/lib/python3.6/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 1, in top-level template code
    {% block doc -%}
  File "/home/mark/anaconda3/envs/flask/lib/python3.6/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 4, in block "doc"
    {%- block html %}
  File "/home/mark/anaconda3/envs/flask/lib/python3.6/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 20, in block "html"
    {% block body -%}
  File "/home/mark/anaconda3/envs/flask/lib/python3.6/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 23, in block "body"
    {% block content -%}
  File "/home/mark/PythonFlaskVertica/app/templates/base.html", line 54, in block "content"
    {% block app_content %}{% endblock %}
  File "/home/mark/PythonFlaskVertica/app/templates/view_stories.html", line 11, in block "app_content"
    {% include '_stories.html' %}
  File "/home/mark/anaconda3/envs/flask/lib/python3.6/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/mark/anaconda3/envs/flask/lib/python3.6/site-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "/home/mark/PythonFlaskVertica/app/templates/_stories.html", line 2, in template
    <div class="col-xs-2"><a href="{{ url_for('view_story_detail', id=story_detail.id)) }}">
  File "/home/mark/anaconda3/envs/flask/lib/python3.6/site-packages/jinja2/environment.py", line 497, in _parse
    return Parser(self, source, name, encode_filename(filename)).parse()
  File "/home/mark/anaconda3/envs/flask/lib/python3.6/site-packages/jinja2/parser.py", line 901, in parse
    result = nodes.Template(self.subparse(), lineno=1)
  File "/home/mark/anaconda3/envs/flask/lib/python3.6/site-packages/jinja2/parser.py", line 875, in subparse
    add_data(self.parse_tuple(with_condexpr=True))
  File "/home/mark/anaconda3/envs/flask/lib/python3.6/site-packages/jinja2/parser.py", line 620, in parse_tuple
    args.append(parse())
  File "/home/mark/anaconda3/envs/flask/lib/python3.6/site-packages/jinja2/parser.py", line 432, in parse_expression
    return self.parse_condexpr()
  File "/home/mark/anaconda3/envs/flask/lib/python3.6/site-packages/jinja2/parser.py", line 437, in parse_condexpr
    expr1 = self.parse_or()
  File "/home/mark/anaconda3/envs/flask/lib/python3.6/site-packages/jinja2/parser.py", line 450, in parse_or
    left = self.parse_and()
  File "/home/mark/anaconda3/envs/flask/lib/python3.6/site-packages/jinja2/parser.py", line 459, in parse_and
    left = self.parse_not()
  File "/home/mark/anaconda3/envs/flask/lib/python3.6/site-packages/jinja2/parser.py", line 470, in parse_not
    return self.parse_compare()
  File "/home/mark/anaconda3/envs/flask/lib/python3.6/site-packages/jinja2/parser.py", line 474, in parse_compare
    expr = self.parse_math1()
  File "/home/mark/anaconda3/envs/flask/lib/python3.6/site-packages/jinja2/parser.py", line 496, in parse_math1
    left = self.parse_concat()
  File "/home/mark/anaconda3/envs/flask/lib/python3.6/site-packages/jinja2/parser.py", line 507, in parse_concat
    args = [self.parse_math2()]
  File "/home/mark/anaconda3/envs/flask/lib/python3.6/site-packages/jinja2/parser.py", line 517, in parse_math2
    left = self.parse_pow()
  File "/home/mark/anaconda3/envs/flask/lib/python3.6/site-packages/jinja2/parser.py", line 528, in parse_pow
    left = self.parse_unary()
  File "/home/mark/anaconda3/envs/flask/lib/python3.6/site-packages/jinja2/parser.py", line 547, in parse_unary
    node = self.parse_postfix(node)
  File "/home/mark/anaconda3/envs/flask/lib/python3.6/site-packages/jinja2/parser.py", line 676, in parse_postfix
    node = self.parse_call(node)
  File "/home/mark/anaconda3/envs/flask/lib/python3.6/site-packages/jinja2/parser.py", line 793, in parse_call
    self.stream.expect('rparen')
  File "/home/mark/anaconda3/envs/flask/lib/python3.6/site-packages/jinja2/lexer.py", line 388, in expect
    next(self)
  File "/home/mark/anaconda3/envs/flask/lib/python3.6/site-packages/jinja2/lexer.py", line 359, in __next__
    self.current = next(self._iter)
  File "/home/mark/anaconda3/envs/flask/lib/python3.6/site-packages/jinja2/lexer.py", line 562, in wrap
    for lineno, token, value in stream:
  File "/home/mark/anaconda3/envs/flask/lib/python3.6/site-packages/jinja2/lexer.py", line 683, in tokeniter
    filename)
jinja2.exceptions.TemplateSyntaxError: unexpected ')'
